flink-table/flink-table-common/src/test/java/org/apache/flink/table/catalog/CatalogTestBase.java

```diff
@@ -1082,6 +1083,64 @@ public void testListPartitionPartialSpec() throws Exception {
 		assertEquals(1, catalog.listPartitions(path1, createAnotherPartitionSpecSubset()).size());
 	}
 
+
+	// ------ table and column stats ------
+
+	@Test
+	public void testGetTableStats_TableNotExistException() throws Exception{
+		catalog.createDatabase(db1, createDb(), false);
+		exception.expect(org.apache.flink.table.api.TableNotExistException.class);
```

Comment:
the `TableNotExistException` is not the class we want. We should use `org.apache.flink.table.catalog.exceptions.TableNotExistException`